data/mods/Magiclysm/Spells/technomancer.json

```diff
@@ -24,8 +24,8 @@
     "range_increment": 1.0,
     "//": "duration is in moves",
     "min_duration": 6000,
-    "max_duration": 10000,
-    "duration_increment": 400
+    "max_duration": 30000,
+    "duration_increment": 1200
```

Comment:
on the second the second thought, 20 minutes seems to be too god, as it gives you more then 1.5 MDR of magus +4 spells. Espasialy cosidering it fast cast time and AOE